### ISSUE TITLE
Add property at caret, please?

### ISSUE THREAD

**Issue by mkahvi** [schultzcole/FVTT-Autocomplete-Inline-Properties#12]
While testing the module, I discovered it adds the properties at the end of the string in input boxes. Could this be changed to do it at caret location?

It was rather annoying to be writing a formula, going back to fill in attributes to discover the attribute was appended to the end of the entire thing, not where I started typing it.

**Comment by schultzcole** [comment]
Yeah, this is something I initially wanted to do but ran into trouble when dealing with selections. I'll look in to adding this.
